lib/dal/src/func/authoring/save.rs

```diff
@@ -40,17 +40,21 @@ pub(crate) async fn update_associations(
                 ))
             }
         },
-        FuncKind::Attribute => match associations {
-            FuncAssociations::Attribute {
-                prototypes,
-                arguments,
-            } => update_attribute_associations(ctx, func, prototypes, arguments).await,
-            invalid => {
-                return Err(FuncAuthoringError::InvalidFuncAssociationsForFunc(
-                    invalid, func.id, func.kind,
-                ))
-            }
-        },
+        FuncKind::Attribute => Ok(()),
+        // don't update attribute associations this way
```

Comment:
we can remove this commented out code but left it for context. Save_func no longer updates Attribute Prototypes